src/import-queue-resolver.ts

```diff
@@ -83,16 +84,18 @@ resolver.define('import', async (req) => {
         await createMRWithCompassYML(project, updatedComponent, groupId);
       }
 
-      const importStatus = 'err' in updatedComponent ? 'fail' : 'success';
-      internalMetrics.counter(`compass.bitbucket.import.end.${importStatus}`).incr();
-      console.log(
-        `GitLab project ${name}:${id} was imported.
+      if ('err' in updatedComponent) {
+        await setFailedRepositoriesToStore(project);
+      } else {
+        internalMetrics.counter(`compass.gitlab.import.end.success`).incr();
+        console.log(
+          `GitLab project ${name}:${id} was imported.
```

Comment:
We should probably not be logging the gitlab project name as it is UGC. I know this is unrelated to your change though